Referenced by issue #125448, (4.2) PyAPI: Assigning dicts as PropertyGroups no longer works in some cases
Referenced by issue #126599, Cannot assign dictionary to existing IdPropertyGroup
